src/app/content/components/Toolbar.tsx

```diff
@@ -99,8 +99,9 @@ const PrintOptWrapper = styled.div`
 `;
 
 // tslint:disable-next-line:variable-name
-const PrintOptions = styled.h3`
+const PrintOptions = styled.span`
```

Comment:
This might need to be a `<button>` but since it's a no-op it doesn't matter for now